napari/utils/_testsupport.py

```diff
@@ -261,6 +261,38 @@ def actual_factory(
                 warnings.warn(msg)
 
 
+@pytest.fixture
+def make_napari_viewer_proxy(make_napari_viewer, monkeypatch):
```

Comment:
Is this used anywhere? If not, did you use it to help debug this PR?

If it's not used, but you think it's useful, you might want to add a comment to clarify that this is useful for debugging and should not be deleted.

Reply:
this is a replacement for `make_napari_viewer` for plugins creators and is referred in my PR to docs https://github.com/napari/docs/pull/7/files 

Currently, plugins during tests do not have warnings about private attribute access, only during actual execution.